File: 프로그래머스/unrated/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.py
def solution(arr):
    result = []
    for num in arr:
        if num >= 50 and num % 2 == 0:
            result.append(num // 2)
        elif num < 50 and num % 2 == 1:
            result.append(num * 2)
        else:
            result.append(num)
    return result

File: 프로그래머스/0/181882. 조건에 맞게 수열 변환하기 1/조건에 맞게 수열 변환하기 1.java
class Solution {
    public int[] solution(int[] arr) {
        int[] answer = new int[arr.length];
        int index = 0;
        for(int i=0;i<arr.length;i++){
            answer[i]=arr[i];
            if(arr[i]>=50 && arr[i]%2==0){
                answer[i] = arr[i]/2;
            }
            else if(arr[i]<50 && arr[i]%2!=0){
                answer[i] = arr[i]*2;
            }
        }
        return answer;
    }
}